.github/workflows/chromatic.ds-react-core-form.yml

```diff
@@ -0,0 +1,16 @@
+name: "ds-react-core/Publish to chromatic"
+on:
+  push:
+    branches:
+      - '**'  # This excludes tags
```

Comment:
This is interesting, it will run tests on *any* push to these packages, even if there's not a pull request.

This has the upside of maintaining always-up-to-date versions of each branch, even feature branches.

However, it also means we will be running a lot more tests (such as people's work-in-progress feature branches that aren't yet PR ready).

I think it will be OK to change this to a pull request trigger.